dir.props

```diff
@@ -120,6 +120,8 @@
       <AdditionalArgs>%(AdditionalArgs) --disable-parallel --infer-runtimes</AdditionalArgs>
     </DnuRestoreDirs>
     <DnuRestoreDirs Include="$(MSBuildThisFileDirectory)src/pkg/deps" />
+    
+    <ShippedNuGetPackage Include="$(PackagesOutDir)/**/*.nupkg" />
```

Comment:
How does this work?  Doesn't this get evaluated before the `nupkgs` are created?

Also, `dir.props` is a dumping ground.  Not every project needs this.  Most don't need it.  This line is only needed in a single project.  Why put this in a common place when only 1 place needs it?